lib/massager_commander.ex

```diff
@@ -0,0 +1,56 @@
+defmodule MassagerCommander do
+  @file_path_root "/ago/aa-work-solve/2022-09-05_SOLV-4646_minnowswim-shopify-historical-import"
+  @single_orders_file "/ago/aa-work-solve/2022-09-05_SOLV-4646_minnowswim-shopify-historical-import/import_massaging/orders_export_06.csv"
+  @all_orders_files [
```

Comment:
This is a very customer-specific way of doing things. This also implies that there is work to rename the files before this program even runs because shopify exports files with numbering starting from 1 (no padding). So... more cumbersome work for a user to do.